lib/dpul_collections/solr.ex

```diff
@@ -149,6 +149,20 @@ defmodule DpulCollections.Solr do
     commit(collection)
   end
 
+  @spec delete_batch(list(), String.t()) ::
+          {:ok, Req.Response.t()} | {:error, Exception.t()} | Exception.t()
+  def delete_batch(ids, collection \\ read_collection()) do
+    ids
+    |> Enum.each(fn id ->
+      Req.post!(
+        update_url(collection),
+        json: %{delete: %{query: "id:#{id}"}}
+      )
+    end)
+
+    commit(collection)
```

Comment:
I like committing, I wonder if we need a soft commit? Otherwise we're looking at a potentially long write lock.

Reply:
Write a follow-up ticket to optimize this deletion, we think maybe there's a delete handler we could use to do them all at once